health-sdk/sdk/src/main/res/values/colors.xml

```diff
@@ -21,25 +21,21 @@
     <color name="ghs_light_04">#8A8A8E</color>
     <color name="ghs_light_05">#565656</color>
     <color name="ghs_light_06">#353535</color>
-    <color name="ghs_light_07">#000000</color>
+    <color name="ghs_light_07">#212121</color>
+    <color name="ghs_success">#B0B347</color>
 
     <color name="ghs_payment_component_caption">@color/ghs_dark_04</color>
     <color name="ghs_bank_selection_bottom_sheet_caption">@color/ghs_dark_03</color>
     <color name="ghs_bank_selection_bottom_sheet_title">@color/ghs_dark_02</color>
+    <color name="ghs_payment_details_background_color">@color/ghs_dark_07</color>
+    <color name="ghs_payment_input_background_color">@color/ghs_dark_05</color>
+    <color name="ghs_payment_input_border_color">@color/ghs_dark_06</color>
```

Comment:
As an example for my review comment, this will always use ghs_dark_06. We may need different color on dark mode.

Reply:
Thank you for pointing this out. I may have misunderstood when discussing how to implement this, but there are corresponding colors declared in `night\colors.xml`, which are being used when we switch to night mode. From what I tested, the correct colors are being used in both day and night mode. 

Am I misunderstanding something?

Reply:
Hi Danila, I'm sorry but I should have used our labels for my comments. This was only a suggestion because it makes the code easier to write and less references needed for the colors. I talked to Alpar about this and we will stick to the way we have for now for Health SDK because we have introduced it every where in our code and it is hard to migrate to the new way. We might use this way in the future in our Merchant SDK. Then, no action is needed from your side. Thank you :-)